Classes/EventStore/AbstractEventSourcedRepository.php

```diff
@@ -84,28 +78,25 @@ public function findByIdentifier($identifier)
     }
 
     /**
-     * @param  AggregateRootInterface $aggregate
+     * @param AggregateRootInterface $aggregate
+     * @param int $expectedVersion
      * @return void
      */
-    public function save(AggregateRootInterface $aggregate)
+    final public function save(AggregateRootInterface $aggregate, int $expectedVersion = null)
```

Comment:
Yes, that's the default. But I might want to *ignore* that constraint or (more commonly) enforce the version to be the same as when I *read* the aggregate (optimistic locking)